docs/input.rst

```diff
@@ -134,6 +134,8 @@ These arguments are also passed as arrays for each iteration, with the same nota
 This example will start by solving a vacuum tokamak, then perturb the pressure profile to solve a finite-beta tokamak, and finally perturb the boundary to solve the finite-beta stellarator.
 If only one value is given, as with ``pert_order`` in this example, that value will be used for all iterations.
 
+If ``pres_ratio`` and ``bdry_ratio`` are not present in the input file, they will be determined automatically during the continuation method.
```

Comment:
and if only a single resolution is given right? otherwise len(inputs)>1 and the automatic continuation method won't be used

Reply:
Yep will update.